.github/comment-ops.yml

```diff
@@ -0,0 +1,7 @@
+commands:
+  label:
+    enabled: true
+  removeLabel:
+    enabled: true
```

Comment:
The use case for these is unclear to me given the large number of maintainers.

Reply:
you can have saved replies that do a few things at once,

e.g. 

```
label web-ui, rfe

reviewer jenkinsci/sig-ux
```
and also this allows non maintainers to setup the labels to save maintainers doing it.

Reply:
> that do a few things at once

Bot commands only makes sense for people who cannot use the built-in features anyway (unless you like spamming watchers).

> this allows non maintainers to setup the labels to save maintainers

Which assumes that people are familiar with how we label things. I already explained in the UX meeting that I've experienced that differently in the subprojects I maintain. Given the large number of active maintainer+ level contributors, the case for this is weak in this repo IMO.

Reply:
>  Given the large number of active maintainer

We don't have what I would call large =/. As one of the (~3) people doing this it would be less work for me if it has been done.